crates/re_viewer_context/src/app_options.rs

```diff
@@ -61,6 +64,10 @@ impl Default for AppOptions {
             // merged in.
             experimental_primary_caching_point_clouds: false,
 
+            // TODO(cmc): default to true for debug/rerun-workspace once minimal features have been
+            // merged in.
```

Comment:
Will be defaulted to `true` later this week when cached range support starts becoming a thing.